assets/files.wxs

```diff
@@ -1826,7 +1826,7 @@
                     </Directory>
                 </Directory>
                 <Directory Id="dir37EBACBC570E49108415C3BA316EB66D" Name="ThreadJob">
-                    <Directory Id="dirCB93DAC4CEB64A8583295B9AC4A79883" Name="1.1.0">
+                    <Directory Id="dirCB93DAC4CEB64A8583295B9AC4A79883" Name="1.1.1">
```

Comment:
Not an issue for this PR.  We should consider not putting the modules in a version folder.  If we have to update the modules, this will make patching very difficult.

Reply:
Filed https://github.com/PowerShell/PowerShell/issues/7236